Assets/Scripts/Scenes/InGame/Player/TempPlayerMove.cs

```diff
@@ -218,17 +219,36 @@ private IEnumerator DecreaseStamina()
 
         private IEnumerator IncreaseStamina()
         {
-            while (_myPlayerStatus.nowPlayerActionState != PlayerActionState.Dash)
+            yield return null;
+            
+            //繝�ヵ繧ｩ繝ｫ繝育憾諷九〒縺ｮ蝗槫ｾｩ
+            if(!_isTiredPenalty)
             {
-                yield return new WaitForSeconds(0.1f);
-                _myPlayerStatus.ChangeStamina(_recoverStamina / 10, "Heal"); 
+                while (_myPlayerStatus.nowPlayerActionState != PlayerActionState.Dash)
+                {
+                    yield return new WaitForSeconds(0.1f);
+                    _myPlayerStatus.ChangeStamina(_recoverStamina / 10, "Heal");
+                }
+
+            }
+            //繧ｹ繧ｿ繝溘リ蛻�ｌ迥ｶ諷九〒縺ｮ蝗槫ｾｩ
+            else縲if(_isTiredPenalty)
+            {
+                yield return new WaitForSeconds(0.5f);//蝗槫ｾｩ髢句ｧ九∪縺ｧ縺ｮ繝ｩ繧ｰ
+
+                while (_myPlayerStatus.nowPlayerActionState != PlayerActionState.Dash)
+                {
+                    yield return new WaitForSeconds(0.1f);
+                    _myPlayerStatus.ChangeStamina(_recoverStaminaOnlyTired / 10, "Heal");
+                }
             }
+
         }
 
         private IEnumerator CountTiredPenalty()
         { 
             _isTiredPenalty = true;
-            yield return new WaitUntil(() => _myPlayerStatus.nowStaminaValue > 10);//�X�^�~�i��10�܂ŉ񕜂���̂�҂�
+            yield return new WaitUntil(() => _myPlayerStatus.nowStaminaValue縲> 10);//繧ｹ繧ｿ繝溘リ縺10縺ｫ縺ｪ繧九∪縺ｧ蝗槫ｾｩ縺吶ｋ縺ｮ繧貞ｾ�▽

```

Comment:
```suggestion
            yield return new WaitUntil(() => _myPlayerStatus.nowStaminaValue >= 100);
```
上のように変更を行えば、スタミナが全回復するまでダッシュができないようになると思います